### PR TITLE
Changed fonts, results_rendering

### PR DESCRIPTION
**Pull request by lennessyy**
It's only some small changes, and hopefully they work (I tested them by copying the JS to the developer console since I don't know how to start a server using Go yet, and they worked on my end so they should be okay)

### CODE REVIEW

File: static/fonts/style.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
```
Comment (ishunyu) [inline static/fonts/style.css:1]: Hey @lennessyy, not sure if I understand why there are so many different BAHNSCHRIFT.woff files? They seems to be the same.

File: static/index.html [hunk 2]
```diff
@@ -1,135 +1,24 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (ishunyu) [inline static/index.html:1]: Appreciate the refactor.

File: static/fonts/style.css (hunk 1 above)
Comment (lennessyy) [reply static/fonts/style.css:1]: I downloaded them online and assumed they they were different weights or something without looking into them deeper. Just deleted the duplicates.

**Comment by ishunyu** [comment]
Can you also remove the duplicate .woff files in `static/fonts`?

**Comment by lennessyy** [comment]
Just deleted! I actually thought I deleted them before but clearly I still don't know how to use github lol (this was actually my first actual pull request)

**Comment by ishunyu** [comment]
No worries, looks good!

**Comment by ishunyu** [comment]
Feel free to merge now.

**Pull request merged**